openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -136,7 +136,10 @@ def create_account_with_params(request, params):
 
     extended_profile_fields = configuration_helpers.get_value('extended_profile_fields', [])
     # Can't have terms of service for certain SHIB users, like at Stanford
-    registration_fields = getattr(settings, 'REGISTRATION_EXTRA_FIELDS', {})
```

Comment:
@iam-mhaseeb why you are not using `extra_fields` here?

Reply:
No specific reason but just realized we can avoid 3 lines change here by just doing 
```
registration_fields = extra_fields
```

Reply:
please don't assign a variable to another. remove the line where we assign value to `registration_fields` and modify these lines.
```
    tos_required = (
        extra_fields.get('terms_of_service') != 'hidden' or
        extra_fields.get('honor_code') != 'hidden'
```